packages/chaire-lib-backend/src/utils/processManagers/ProcessManager.ts

```diff
@@ -12,6 +12,8 @@ import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
 import ProcessUtils from './ProcessUtils';
 
 const PID_DIRECTORY = 'pids';
+const DEFAULT_LOG_FILE_COUNT = 3; // 3 files
```

Comment:
Shouldn't those be removed in the last commit, now that we have default in the config file???

Reply:
Can we combine those constant somehow? Can the trRouting default config information use those constant ?

Reply:
Moved to the config file